packages/react-components/src/unstable/index.ts

```diff
@@ -123,31 +123,37 @@ export {
 } from '@fluentui/react-switch';
 export type { SwitchOnChangeData, SwitchProps, SwitchSlots, SwitchState } from '@fluentui/react-switch';
 
-export {
-  Tab,
-  /* eslint-disable-next-line deprecation/deprecation */
-  tabClassName,
-  tabClassNames,
-  useTab_unstable,
-  useTabStyles_unstable,
-  renderTab_unstable,
-  TabList,
-  /* eslint-disable-next-line deprecation/deprecation */
-  tabListClassName,
-  tabListClassNames,
-  useTabList_unstable,
-  useTabListStyles_unstable,
-  renderTabList_unstable,
-} from '@fluentui/react-tabs';
 export type {
-  TabValue,
+  TabContentRect,
   TabProps,
+  TabSlots,
   TabState,
+  TabValue,
+  RegisterTabData,
+  RegisterTabEventHandler,
```

Comment:
Let's try to be consistent with type names, for example `TabRegisterData` makes more sense. If these types are not designed to be used by customers let's do not re-export them for now.

Reply:
My instinct is to name things with adjective before noun - so if there were other types of tab data something like position data or something it would be PositionTabData.  

However, if our naming convention is to use the component name like scope or namespace, then I'm happy to rename.